spec/student_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../classes/student'
+
+describe Student do
```

Comment:
- [ Required ] Please consider adding a unit test for the `play_hooky` and `classroom` methods in your class `Student`, the project requirements state that all the methods in your classes need to be tested.

## Project requirements
![Screenshot from 2023-03-30 23-55-19](https://user-images.githubusercontent.com/16392046/228962573-8389fc09-4ce6-42de-b1af-aefd25f2d036.png)